app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
+
+  def recent_posts
+    posts.order(created_at: :desc).last(3)
+  end
```

Comment:
- [ ] with the `.last` method you're getting the last 3 items in the array which in this case are the 3 oldest posts, you should however try to get the 3 most recent posts.